bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -14,7 +14,14 @@ struct {
 	__type(key, u32);
 	__type(value, u64);
 	__uint(max_entries, MAP_SIZE);
-} pid_time SEC(".maps");
+} pid_time_map SEC(".maps");
+
+struct {
+	__uint(type, BPF_MAP_TYPE_HASH);
+	__type(key, u32);
+	__type(value, u32);
+	__uint(max_entries, MAP_SIZE);
+} pid_tgid_map SEC(".maps");
```

Comment:
This naming is problematic to me. See where we access this map in the code using `prev_pid` as the key. We should take this opportunity to use sensible names here.

Reply:
@vimalk78 and I had a lengthy discussion about this in a PR a week or so ago and it's very easy to get confused

Reply:
What about `pid_to_tgid_map`?
We just want to get the `tgid` of a give `pid`, which in case will be the `prev_pid`